cli/mender-artifact/main.go

```diff
@@ -272,12 +296,18 @@ func getCliContext() *cli.App {
 			Usage: "Include `FILE` in payload in the augment section. Can be given more than once.",
 		},
 		compressionFlag,
+		//////////////////////
+		// Sotware versions //
+		//////////////////////
+		softwareVersionNoDefault,
+		softwareVersionName,
+		softwareVersionValue,
```

Comment:
What about `--software-filesystem`? This is supposed to be available for `module-image` as well.

Reply:
Added